akka-actor-typed/src/main/scala/akka/actor/typed/internal/adapter/ActorAdapter.scala

```diff
@@ -102,7 +103,21 @@ import scala.annotation.switch
 
   private def handleMessage(msg: T): Unit = {
     try {
-      next(Behavior.interpretMessage(behavior, ctx, msg), msg)
+      val c = ctx
```

Comment:
why do we need to store this as a val? this looks to be lazy but never replaced

Reply:
avoiding duplicate calls to `def ctx` avoiding null check, yeah probably premature